apps/admin/src/pages/Proposals.tsx

```diff
@@ -102,7 +96,7 @@ export function Proposals() {
   };
 
   useEffect(() => {
-    filterProposals();
+    filterProposals({});
```

Comment:
Would it be worthwhile to check first if there's an existing filter before firing filter on page load?